PlayerInfo/doc/PlayerInfoPlugin.md

```diff
@@ -0,0 +1,467 @@
+<!-- Generated automatically, DO NOT EDIT! -->
+<a name="head.Player_Info_Plugin"></a>
```

Comment:
Pls remove this md file. Docs are generated under docs folder at https://github.com/rdkcentral/rdkservices/blob/sprint/2208/docs/api/PlayerInfoPlugin.md